src/BenchmarksServer/Startup.cs

```diff
@@ -211,28 +211,12 @@ private static string CloneAndRestore(string path, ServerJob job)
 
             Debug.Assert(benchmarksDir != null);
 
-            // Modify all global.json to reference source dirs
```

Comment:
Created a separate issue to update BenchmarksServer source dependencies feature for MSBuild: https://github.com/aspnet/benchmarks/issues/185